ui/tests/integration/components/shamir-modal-flow-test.js

```diff
@@ -0,0 +1,103 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find, pauseTest, click } from '@ember/test-helpers';
```

Comment:
looks like you'll need to remove `pauseTest` and `click` to get the tests to pass in circleCi: https://app.circleci.com/pipelines/github/hashicorp/vault/9569/workflows/866d34ee-1f71-4f78-94ea-173354ef632a/jobs/62858/steps